test/e2e/tests/request-queuing/multi-dapp-sendTx-revokePermission.spec.js

```diff
@@ -81,40 +75,53 @@ describe('Request Queuing for Multiple Dapps and Txs on different networks revok
         await openDapp(driver, undefined, DAPP_ONE_URL);
 
         // Connect to dapp 2
-        await driver.findClickableElement({ text: 'Connect', tag: 'button' });
-        await driver.clickElement('#connectButton');
+        await driver.clickElement({ text: 'Connect', tag: 'button' });
 
-        await driver.delay(regularDelayMs);
+        await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
 
-        await switchToNotificationWindow(driver, 4);
-
-        await driver.clickElement({
+        await driver.clickElementAndWaitForWindowToClose({
           text: 'Connect',
           tag: 'button',
         });
 
         // Dapp 1 send tx
         await driver.switchToWindowWithUrl(DAPP_URL);
-        await driver.delay(largeDelayMs);
+        await driver.findElement({
+          css: '[id="chainId"]',
+          text: '0x1',
+        });
         await driver.clickElement('#sendButton');
 
         await driver.waitUntilXWindowHandles(4);
         await driver.delay(3000);
 
         // Dapp 2 send tx
         await driver.switchToWindowWithUrl(DAPP_ONE_URL);
-        await driver.delay(largeDelayMs);
+        await driver.findElement({
+          css: '[id="chainId"]',
+          text: '0x53a',
+        });
         await driver.clickElement('#sendButton');
+        await driver.waitUntilXWindowHandles(4);
 
         // Dapp 1 revokePermissions
         await driver.switchToWindowWithUrl(DAPP_URL);
-        await driver.clickElement('#revokeAccountsPermission');
+        await driver.findElement({
+          css: '[id="chainId"]',
+          text: '0x1',
+        });
+        await driver.assertElementNotPresent({
+          css: '[id="chainId"]',
+          text: '0x53a',
+        });
```

Comment:
to avoid a race condition where dapp doesn't have the correct network (see [here](https://output.circle-artifacts.com/output/job/6e0051a3-8a15-416d-9794-04eb59f28a10/artifacts/7/test-artifacts/undefined/Request%20Queuing%20for%20Multiple%20Dapps%20and%20Txs%20on%20different%20networks%20revokePermissions%20should%20close%20transaction%20for%20revoked%20permission%20of%20eth_accounts%20but%20show%20queued%20tx%20from%20second%20dapp%20on%20a%20different%20network./test-failure-dom.html))


![Screenshot from 2024-10-09 14-34-56](https://github.com/user-attachments/assets/bbf38366-d380-4662-80d2-c876d8b3044d)